images/kindnetd/go.mod

```diff
@@ -1,49 +1,64 @@
 module sigs.k8s.io/kind/images/kindnetd
 
-go 1.21
+go 1.22.0
```

Comment:
this requires 1.22 but kind is still in 1.21, should we bump kind or make kindnet go version independent

Reply:
the runc issue is recent and unusual, we should leave them the same for now and bump it, started on that.

Reply:
done in #3676